rasa/nlu/constants.py

```diff
@@ -35,7 +35,14 @@
     RESPONSE_ATTRIBUTE: "response_dense_features",
 }
 
-SPACY_DOCS = {TEXT_ATTRIBUTE: "spacy_doc", RESPONSE_ATTRIBUTE: "response_spacy_doc"}
+SPACY_DOCS = {
+    TEXT_ATTRIBUTE: "text_spacy_doc",
+    RESPONSE_ATTRIBUTE: "response_spacy_doc",
+}
+HF_TRANSFORMERS_DOCS = {
```

Comment:
@Ghostvv Should we make this `LM_DOCS` or `LANGUAGE_MODEL_DOCS`? I don't want to make them specific to `HF` or `HF_TRANSFORMERS` here because as discussed these docs can ideally be set by another `NLP` component in the future.
Also, these docs contain the very basic attributes of any model - `token_ids`, `tokens` , `sequence features` and `sentence features`. Nothing specific to HFTransformers. What do you think?

Reply:
sure, `LANGUAGE_MODEL_DOCS` sounds good to me